hsds/util/fileClient.py

```diff
@@ -108,7 +108,20 @@ def _file_stats_increment(self, counter, inc=1):
             log.error(f"unexpected inc for file_stats: {inc}")
             return
 
-        file_stats[counter] += inc
+    def getURIFromKey(self, key, bucket=None):
```

Comment:
Might be less confusing to have the version here in `fileClient.py` and the version in `azureBlobClient.py` be merged with an argument to decide the exact uri returned.

Reply:
I'll come back to this later - will need some changes for the azure uri strings.